cdk/typescript/test/the-lambda-power-tuner.test.ts

```diff
@@ -0,0 +1,17 @@
+import { Template } from 'aws-cdk-lib/assertions';
+import * as cdk from 'aws-cdk-lib';
+import * as TheLambdaPowerTuner from '../lib/the-lambda-power-tuner-stack';
+
+test('SAR Application Created', () => {
```

Comment:
I assume this runs smoothly when you execute `npm run test`, right?

I cannot think of additional tests to run here for now, but things will definitely change if/once we drop SAR support :)

Reply:
Yes, I have tested this out. It works smoothly.